Chat/src/app/utils/utils.ts

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { GUID_FOR_INITIAL_TOPIC_NAME } from './constants';
+
+declare let __BUILDTIME__: string; // Injected by webpack
+export const getBuildTime = (): string => __BUILDTIME__;
+
+export function getChatSDKVersion(): string {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  return require('../../../package.json')['dependencies']['@azure/communication-chat'];
```

Comment:
nit: this should be done by webpack to ensure the live deployed app doesn't need access to the pacakge.json

Reply:
Created ADO bug
https://skype.visualstudio.com/SPOOL/_workitems/edit/2691370